app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1982,6 +1989,40 @@ class BrowserTabFragment :
         viewModel.onUserClickCtaSecondaryButton()
     }
 
+    private fun showBackNavigationHistory(history: ShowBackNavigationHistory) {
+        activity?.let { context ->
+            BottomSheetDialog(context, com.duckduckgo.mobile.android.R.style.NavigationHistoryDialog).also {
+                it.setContentView(R.layout.navigation_history_popup_view)
```

Comment:
nit: When we upgraded AGP, I changed most of these to `import com.duckduckgo.mobile.android.R as CommonR`. Not a big deal though.